src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -993,8 +996,14 @@ static void window_top_toolbar_paint(rct_window* w, rct_drawpixelinfo* dpi)
         y = w->y + window_top_toolbar_widgets[WIDX_NETWORK].top + 0;
         if (widget_is_pressed(w, WIDX_NETWORK))
             y++;
-        imgId = SPR_SHOW_GUESTS_ON_THIS_RIDE_ATTRACTION;
-        gfx_draw_sprite(dpi, imgId, x, y, 0);
+        imgId = SPR_G2_MULTIPLAYER_TOOLBAR;
+        gfx_draw_sprite(dpi, imgId, x - 2, y - 2, 0);
+        gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
+        imgId = (network_is_desynchronised() ? SPR_G2_MULTIPLAYER_DESYNC : SPR_G2_MULTIPLAYER_SYNC);
```

Comment:
This should be guarded for network availability, e.g. through:
```
#ifndef DISABLE_NETWORK
...
#endif
```

Reply:
The rest of the toolbar code doesn't check for `#ifndef DISABLE_NETWORK`, so I'm unsure if you mean the _entirety_ of the toolbar code involving network options should be wrapped in this.

However, it _is_ indirectly protected; the toolbar button is only available if `network_get_mode()` returns `NETWORK_MODE_CLIENT` or `NETWORK_MODE_SERVER` (see line 789), and if `DISABLE_NETWORK` is set, `network_get_mode()` will _always_ return `NETWORK_MODE_NONE`. This same outcome applies to the Chat button as well.

Reply:
Please have a look at the [CI output for the networkless build](https://travis-ci.org/OpenRCT2/OpenRCT2/jobs/489833657).

Reply:
That was because two of the functions I added to `openrct2/network/Network.cpp` were missing from its `DISABLE_NETWORK` section. I've resolved that in the latest commit.